Content.Server/Health/BodySystem/BodyPart/BodyPart.cs

```diff
@@ -114,36 +114,51 @@ public class BodyPart
         [ViewVariables]
         public List<Mechanism> Mechanisms => _mechanisms;
 
-        public BodyPart(){}
+        public BodyPart() { }
 
         public BodyPart(BodyPartPrototype data)
         {
             LoadFromPrototype(data);
         }
 
-
-
+        public bool CanAttachBodyPart(BodyPart toBeConnected)
+        {
+            return _surgeryData.CanAttachBodyPart(toBeConnected);
+        }
 
         /// <summary>
-        ///     Attempts to add a Mechanism. Returns true if successful, false if there was an error (e.g. not enough room in BodyPart). Use InstallDroppedMechanism if you want to easily install an IEntity with a DroppedMechanismComponent.
+        ///     Returns whether the given <see cref="Mechanism">Mechanism</see> can be installed on this BodyPart.
```

Comment:
Unless you want something special to appear in the doc link, just `<see cref="Mechanism"/>` will have the same effect here - since that automatically prints the name of the member you're linking to.

Reply:
Fixed this and did the rest of the summaries while I was at it.